### DISCUSSION TITLE
First-time install on Mac  but get 'Error' msg

### DISCUSSION THREAD

**Discussion by takeitothegrill** [Homebrew/discussions#1430] [Getting started]
1. Installed Homebrew on macOS 10.15.7 successfully.

2. Attempted  'sudo brew install libusb python' in terminal.

3. Received the following msg after entering password:

Error: Running Homebrew as root is extremely dangerous and no longer supported.
As Homebrew does not drop privileges on installation you would be giving all
build scripts full access to your system.

4. QUESTION: Is this happening because my local user account may have Administrator privileges?

**Comment by gromgit** [comment]
> QUESTION: Is this happening because my local user account may have Administrator privileges?

No. It's because the Homebrew installer specifically set things up so that you can `brew install` without root.

**Reply by takeitothegrill** [reply]
Heard.
